src/router.js

```diff
@@ -156,8 +163,15 @@ const router = createRouter({
         {
           path: 'run/:id',
           name: 'PacbioRunShow',
-          component: PacbioRunShow,
-          props: true,
+          component: FlaggedFeatureView,
+          props: (route) => ({
+            feature: 'dpl_1113_run_well_aliquots',
```

Comment:
I think these should have separate feature flags as they are separate features.

Reply:
It is Pacbio runs page and run edit page. Do you think they need to be separate feature flags? I was under the impression that  they should be the same as they both are related to runs?

Reply:
Hmm im not sure. My thinking was if there is a bug with the runs page we dont want to also have to turn off the runs/create page if there is nothing wrong.

Reply:
Oh, ok. I think then I will separate it. 👍

Reply:
Done 👍